src/main/java/org/oasis_eu/portal/model/store/OrganizationHistory.java

```diff
@@ -0,0 +1,39 @@
+package org.oasis_eu.portal.model.store;
```

Comment:
here, `store` refers to the catalog, not the right place for organization history stuff :)